src/cityreader/cityreader.py

```diff
@@ -1,11 +1,19 @@
+import csv
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, latitude, and longitude.
 
 
+class City():
+    def __init__(self, name, latitude, longitude):
+        self.name = name
+        self.latitude = latitude
+        self.longitude = longitude
+
```

Comment:
you probably want the method __str__ or __repr__ to read the city info name latitude and longitude